addons/test_mass_mailing/models/__init__.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
```

Comment:
About the commit message: english needs fixing.

Reply:
Done